.adaptable/config.schema.json

```diff
@@ -83,7 +87,8 @@
             "type": "string"
         },
         "startCommand": {
-            "type": "string"
+            "type": "string",
+            "tags": ["go", "nodejs", "php", "python"]
```

Comment:
Do we not want to allow override of the entrypoint/run command in Dockerfile if the user shares a Dockerfile between dev and prod?

Reply:
Not implementing now, per discussion.